test/dynamo/test_unspec.py

```diff
@@ -172,8 +172,7 @@ def fn(x):
         res2 = opt_fn(x)
         self.assertTrue(same(res1, res2))
 
-    # TypeError: zeros(): argument 'size' (position 1) must be tuple of SymInts, not FakeTensor
-    @unittest.expectedFailure
+    @torch._dynamo.config.patch("specialize_int_float", True)
```

Comment:
This diff makes no sense, this is `test_unspec.py`, it's specifically about testing `specialize_int_float=False`